twitter.html

```diff
@@ -0,0 +1,210 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>twitter</title>
+<link rel="stylesheet" href="twitter.css">
+</head>
+
+<body>
+    <div class="container">
+
+<header>
+
+    <div id="main-header">
+        <strong>Home</strong>
+        <img class="star" src="https://www.svgrepo.com/show/529944/star-fall-minimalistic-2.svg" width="50" alt="star">
+    </div>
+
+    <div class="sub-header">
+        <p contenteditable>What's happening?</p>
+       <img class="profile-pic" src="https://www.svgrepo.com/show/498301/profile-circle.svg" width="50" alt="profile-pic">
+    
+      
+       <div class="post">
+    
+        <button><strong>Tweet</strong></button>
+
+           <img class="picture-icon" src="https://www.svgrepo.com/show/438208/image-frame-photo-tagged-picture-icon.svg" width="50" alt="picture symbol">
+           <img class="gif" src="https://www.svgrepo.com/show/520750/gif.svg" width="50" alt="gif symbol">
+           <img class="stats-bar" src="https://www.svgrepo.com/show/533359/chart-simple-horizontal.svg" width="50" alt="stats-bar">
+           <img class="smiley-face" src="https://www.svgrepo.com/show/441173/emoji-smile.svg" width="50" alt="smiley-face"> 
+           <img class="calender" src="https://www.svgrepo.com/show/357437/calender.svg" width="50" alt="calender">
+        </div>
+    </div>
+
+</header>
+
+
+
+
+<div class="left-tab">
+
+    <img class="twitter-logo" src="https://www.svgrepo.com/show/183608/twitter.svg" width="50" alt="twitter-logo">
+    
+    <div class="home">
+        <img class="bird-house" src="https://www.svgrepo.com/show/387796/home.svg" width="50" alt="home">
+        <strong><a href="./page2.html">Home</a></strong>
+    </div>
+
+    <div>
+        <img class="explore" src="https://www.svgrepo.com/show/42960/hashtag.svg" width="50" alt="explore">
+        <strong><a href="./page2.html">Explore</a></strong>
+    </div>
+
+    <div>
+        <img class="notifications" src="https://www.svgrepo.com/show/361809/notification.svg" width="25" alt="notification">
+        <strong><a href="./page2.html">Notifications</a></strong>
+    </div>
+
+    <div>
+        <img class="messages" src="https://www.svgrepo.com/show/408760/message-mail-email-letter-envelope.svg" width="50"
+        alt="messages">
+        <strong><a href="./page2.html">Messages</a></strong>
+    </div>
+
+    <div>
+        <img class="bookmarks" src="https://www.svgrepo.com/show/379820/bookmark.svg" width="50" alt="bookmarks">
+        <strong><a href="./page2.html">Bookmarks</a></strong>
+    </div>
+
+    <div>
+        <img class="lists" src="https://www.svgrepo.com/show/18393/list.svg" width="50"alt="lists">
+        <strong><a href="./page2.html">Lists</a></strong>
+    </div>
+
+    <div>
+        <img class="profile" src="https://www.svgrepo.com/show/318358/profile.svg" width="50" alt="profile">
+        <strong><a href="./page2.html">Profile</a></strong>
+    </div>
+
```

Comment:
I downloaded and saw that the status bar to post something had and outline when it write something.So, I think you can do outline none in input text

Reply:
Perfect, didn't know you could get rid of that, thank you